scripts/mixins/warders_cop.lua

```diff
@@ -0,0 +1,51 @@
+-----------------------------------
+-- Mixin for Warder NMs in CoP
+-----------------------------------
+require('scripts/globals/mixins')
+require('scripts/globals/status')
+-----------------------------------
+
+g_mixins = g_mixins or {}
+
+g_mixins.warders_cop = function(mob)
+
+    mob:addListener('COMBAT_TICK', 'WARDER_CTICK', function(mob)
+        local battletime = mob:getBattleTime()
+        local changeTime = mob:getLocalVar('changeTime')
+        local initiate = mob:getLocalVar('initiate')
+        local warder = mob:getLocalVar('warder')
+        local electro = mob:getLocalVar('electro')
+        local reactive = mob:getLocalVar('reactive')
+        local ID = zones[mob:getZoneID()]
+
+        if battletime - initiate >= 27 and
+           warder <= 1
+        then -- warder signals to the others before using abilities
+            mob:showText(mob, ID.text.BEEP_BEEP)
+            mob:setLocalVar('initiate', battletime)
+        elseif battletime - changeTime >= 30 and
+               warder >= 1
+        then -- shortly after each will use the same ability at the same time
+            mob:showText(mob, ID.text.BEEP_CLICK_WHIR)
+            mob:setLocalVar('initiate', battletime)
+            mob:setLocalVar('changeTime', battletime)
+            if electro == 1 then
+                mob:useMobAbility(561) -- use Electromagnetic Field
+                mob:setLocalVar('electro', 0)
+            elseif reactive == 1 then
+                mob:useMobAbility(562) -- use Reactive Armor
+                mob:setLocalVar('reactive', 0)
+            end
+        end
+    end)
+
+    mob:addListener('WEAPONSKILL_STATE_ENTER', 'WS_START_MSG', function(mob, skillID)
```

Comment:
`function(mob, skillID)` - This shadows the mob parameter of the original function.  To resolve, the pattern we've been using for the lower function is to change it to `function(mobArg, skillID)` and replace the listener function references of mob to mobArg